.github/workflows/build.yml

```diff
@@ -159,7 +159,7 @@ jobs:
       - name: Run component tests against ${{ matrix.multitarget }}
         if: ${{ matrix.multitarget == 'uwp' || matrix.multitarget == 'wasdk' }}
         id: test-platform
-        run:  vstest.console.exe ./tooling/**/CommunityToolkit.Tests.${{ matrix.multitarget }}.build.appxrecipe /Framework:FrameworkUap10 /logger:"trx;LogFileName=${{ matrix.multitarget }}.trx" /Blame
+        run:  vstest.console.exe ./tooling/**/CommunityToolkit.Tests.${{ matrix.multitarget }}.build.appxrecipe /Framework:FrameworkUap10 /logger:"trx;LogFileName=${{ matrix.multitarget }}.trx" /Blame:"CollectDump;DumpType=Full;CollectHangDump;TestTimeout=30m;HangDumpType=Full" /Diag:"${{ github.workspace }}/vstest-diagnostic-log.txt"
```

Comment:
Not sure where the Blame dump gets created, so we may need to figure that out so we can upload it to the artifacts.

Reply:
The plan was to see if they'd get picked up by the code we already have for collecting crash dumps, and to investigate further if it didn't work.

Reply:
Looks like the `/Diag` logs were uploaded as artifacts, but no crash dumps. Investigating...

Reply:
Crash dumps are being uploaded as artifacts now